src/Models/AbstractContentModel.php

```diff
@@ -42,15 +43,27 @@ abstract class AbstractContentModel extends Model
     protected $appends = ['status'];
 
     protected $casts = [
-        'publish_at' => 'datetime',
-        'expire_at' => 'datetime',
+        'publish_at'    => 'datetime',
+        'expire_at'     => 'datetime',
+        'contents'      => 'json',
     ];
 
     protected $dispatchesEvents = [
         'creating' => ContentCreating::class,
         'updating' => ContentUpdating::class,
     ];
 
+    public function shortDescription($length = 50, $omission = '...'): string
+    {
+        foreach ($this->contents['content'] as $key => $value) {
+            if ($value['type'] === 'paragraph') {
+                return Str::of($value['content'][0]['text'])->take($length) . $omission;
+            }
+        }
+
+        return '...';
```

Comment:
Should return `$omission` here?

Also, `shortDescription` should be a proper attribute